specification_situation.md

```diff
@@ -1,43 +1,54 @@
 <base target="_blank">
 
-The specification of Client Hints is divided between different standardization
-bodies and different standards.  To make matters worse, since the feature does
-not yet have multiple browser implementation support, some of the specification
-for the feature lives in PRs on the relevant standards.
+The specification of Client Hints is divided between several different
+standards.  To make matters worse, since Client Hints does not yet have support
```

Comment:
Nitpicking, but we're not starting from "worse". It's not uncommon for a specification to have requirements and hooks across different specs. I'd just note that that some of the spec integrations are still waiting to be approved and merged, conditional on a second implementer other than Chromium.

Reply:
Yeah, that's fair